itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/QuarkusIntegrationTests.java

```diff
@@ -41,6 +42,7 @@
 	CreateNewProjectTest.class,
 	CreateNewQuarkusConfigurationMavenTest.class,
 	CreateNewQuarkusConfigurationGradleTest.class,
+	EnvironmentTest.class,
```

Comment:
I would rename it to EnvironmentVariablesTest(and also the abstract class). EnvironmentTest is very general :(

Reply:
done